create_termsheet.py

```diff
@@ -0,0 +1,98 @@
+"""
+create_termsheet is a command line tool that can be used to render
+a termsheet based on the Origin termsheet template and a json file
+based on the Airbrush data standard
+
+Usage:
+    python create_termsheet.py <json_input_file> <docx_output_file>
+
+Args:
+    <json_input_file> is the path to a json file in Airbrush format
+    <docx_output_file> is the path to the rendered termsheet to create
+"""
+import argparse
+from pathlib import Path
+import json
+from docxtpl import DocxTemplate
+
+BASE_DIR = Path(__file__).parent
+TERMSHEET_TEMPLATE_PATH = BASE_DIR / "Machine-Readable Termsheet.docx"
+
+ARGNAME_INPUT_JSON_PATH = "json_input_file"
+ARGNAME_OUTPUT_DOCX_PATH = "docx_output_file"
+
+
+def validate_file_type(parser, file_path, ext, must_exist=True):
+    """
+    Validates file path string passed to an argparse parser. Checks that string has a valid
+    extension and is a file that exists if must_exist = True. Raises validation errors
+    if any of the conditions fail.
+
+    Args:
+        parser: An argparse.ArgumentParser instance used to raise validation errors
+        file_path: The file path string to be validated
+        ext: The file extenion to validate the file path against. In the format '.*'
+        must_exist: Boolean for whether to check existence of file in validation. Defaults to True
+
+    Returns:
+        str | None: The string passed to it if its valid, otherwise None
+    """
+    path = Path(file_path)
+    if must_exist and not (path.exists() and path.is_file):
+        parser.error(f"The file {file_path} does not exist")
+    elif not path.suffix == ext:
```

Comment:
think the standard way of doing this condition is `elif path.suffix != ext:` but yours still works

Reply:
Makes sense, that's clearer. Will update